<launch>
  <arg name="set_port" default="/dev/ttyTHS1"/>
  <node pkg="omo_r1mini_bringup" type="motor_vs.py" name="omo_r1mini_motor_setting" output="screen">
      <param name="port" value="$(arg set_port)"/>
      <param name="baud" value="115200"/>
      <param name="modelName" value="r1mini"/>
  </node>
  <rosparam command="load" file="$(find omo_r1mini_bringup)/config/robot_config.yaml" />
</launch>